lib/private/help-exec-machine-instance.js

```diff
@@ -806,7 +806,7 @@ module.exports = function helpExecMachineInstance (liveMachine) {
       } catch (e) {
 
         // Split the error we got into lines, and remove the first one (the actual error message)
-        e.stack = e.stack.split('\n');
+        e.stack = stripStackMessage(e).split('\n');
         var msg = e.stack.shift();
```

Comment:
@mikermcneil I think you were commenting on outdated source.  This is what was causing the tests to fail, and was fixed in https://github.com/node-machine/machine/pull/21/commits/639f66788799da88934b0f26afb46667d23ac488